frontend/Python/frontend.py

```diff
@@ -448,6 +448,20 @@ def importer(self, model, *args, **kwargs) -> List[Graph]:
         model_opt(*args, **kwargs)
         return self._imported_graphs
 
+    def importer_by_export(self, module: torch.nn.Module, *args) -> List[Graph]:
+        """
+        imports the provided model as MLIR module and float parameters by `torch.export.export`.
```

Comment:
Please add the related links to docstring to describe this change:
- https://docs.pytorch.org/docs/stable/export.html
- https://github.com/pytorch/pytorch/issues/128334